### ISSUE TITLE
Compiling under VS 2015 Update 3

### ISSUE THREAD

**Issue by manu-silicon** [jbevain/cecil#283]
The presence of the `project.json` and its corresponding `project.lock.json` are causing the build of Mono.Cecil to fail from VisualStudio. The error I'm getting is:

```
1>C:\Program Files (x86)\MSBuild\Microsoft\NuGet\Microsoft.NuGet.targets(140,5): error : Your project is not referencing the ".NETFramework,Version=v3.5" framework. Add a reference to ".NETFramework,Version=v3.5" in the "frameworks" section of your project.json, and then re-run NuGet restore.
```

Getting rid of both files, I'm able to build under VS but the build is always marked as failing with the following error:

```
An error occurred while reading file 'C:\work\Mono.Cecil\project.json': Could not find file 'C:\work\Mono.Cecil\project.json'.
```

Building from the command line is working just fine.

**Comment by jbevain** [comment]
Eurk. Thanks for the heads-up.

**Comment by jbevain** [comment]
Should be fixed in 9636b243dda40914e8de7dfccb5f54bc14c8b468. Thanks!
